<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestión de Productos</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
    background-color: #4CAF50;
    color: white;
    font-size: 16px;
    padding: 10px 20px;
    border-radius: 5px;
}
QTableWidget {
    border: 1px solid #ddd;
    font-size: 14px;
}
QLabel {
    font-weight: bold;
    font-size: 14px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>1021</width>
      <height>67</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="buscarProducto">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Buscar producto</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>1021</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableWidget" name="tabla_productos">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>250</height>
        </size>
       </property>
       <property name="columnCount">
        <number>9</number>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>145</number>
       </attribute>
       <column>
        <property name="text">
         <string>ID Producto</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Nombre</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Categoría</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Ubicación </string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Medida</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Cantidad</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Precio</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Total</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Acciones</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lbl_ultima_modificacion">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>540</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Última modificación: --/--/---- --:--</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>1011</width>
      <height>47</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnAgregar">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>49</height>
        </size>
       </property>
       <property name="text">
        <string>Agregar Producto</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnImprimir">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>49</height>
        </size>
       </property>
       <property name="text">
        <string>Imprimir</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1040</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
